sql/wsrep_server_state.h

```diff
@@ -36,6 +36,8 @@ class Wsrep_server_state : public wsrep::server_state {
   static void destroy();
   static Wsrep_server_state &instance() { return *m_instance; }
 
+  static bool has_instance() noexcept { return m_instance != nullptr; }
```

Comment:
m_instance is created in init_once method. Would bool initialized() be the better name for this?

Reply:
This method will also return false during shutdown, after the instance is already destructed. So I'm not sure about `initialized`, as technically, it was already initialized before.

Reply:
Wsrep_server_state::init_once() seems to be complementary function to Wsrep_server_state::destroy()., so init_once -> initialized -> destrony -> not initialized -> init_once -> initialized.

Anyway, this is just a matter of naming convention and in my opinion has_instance() exposes internal detais of the object implementation.

Reply:
Changed the name.